README.md

```diff
@@ -1,8 +1,8 @@
-# Hawkular Docker images
+# Hawkular APM Docker images
 
-This is a set of Docker images related to Hawkular:
+This is a set of Docker images related to [Hawkular APM (Application Performance Management)](https://github.com/hawkular/hawkular-apm):
 
-- [hawkular-aio](https://registry.hub.docker.com/u/jboss/hawkular-aio) Hawkular All-In-One, runs embedded Cassandra and all you need in a single container
 - [hawkular-apm-server](https://registry.hub.docker.com/u/jboss/hawkular-apm-server) Hawkular APM, runs a standalone Application Performance Management server with embedded Elasticsearch (by default)
```

Comment:
It is not a change in this PR but this link redirects me to my page or root url of the hub.

This link should work:
https://hub.docker.com/r/jboss/hawkular-apm-server/